<launch>
  <!-- use sim time >
  <param name="use_sim_time" value="true"/-->

  <!-- Robot pose ekf -->
  <node pkg="robot_pose_ekf" type="robot_pose_ekf" name="robot_pose_ekf" output="screen">
    <param name="output_frame" value="odom"/>
    <param name="base_footprint_frame" value="base_link"/>
    <param name="base_frame" value="base_link"/>
    
    <param name="freq" value="10.0"/>
    <param name="sensor_timeout" value="1.0"/>
    <param name="odom_used" value="true"/>
    <param name="imu_used" value="ture"/>
    <param name="vo_used" value="false"/>
    <remap from="odom" to="robomuse_diff/odom" />
    <remap from="imu_data" to="imu_in" />
 
   
  </node>



  <!--test test-name="test_robot_pose_ekf" pkg="robot_pose_ekf" type="test_robot_pose_ekf" time-limit="120" /-->
</launch>
